pgcli/main.py

```diff
@@ -103,16 +103,13 @@ def cli(database, user, password, host, port):
             try:
                 _logger.debug('sql: %r', document.text)
                 res = pgexecute.run(document.text)
-                output = []
```

Comment:
That exists because https://github.com/amjith/pgcli/blob/master/pgcli/pgexecute.py#L111 can return results for more than one table. 

eg: This command will return the table details for multiple tables. 

```
django_app> \d django*
```

Can you elaborate why you say it is buggy?